### PR TITLE
Issue #48: added self serve instructions in response for adobe employees

### PR DESCRIPTION
**Pull request by stevengill**
Added handbook link in response

### CODE REVIEW

File: checker/checker.js [hunk 1]
```diff
@@ -291,7 +291,7 @@ function action_required (ow, args) {
...
       title: 'Sign the Adobe CLA!',
-      summary: 'No signed agreements were found. Please [sign the Adobe CLA](http://opensource.adobe.com/cla.html)! Once signed, close and re-open your pull request to run the check again.\n\n If you are an Adobe employee, you do not have to sign the CLA. Instead contact Adobe\'s Open Source Office about the failure by mentioning them on the pull request with **@adobe/open-source-office** or via email <grp-opensourceoffice@adobe.com>.'
+      summary: 'No signed agreements were found. Please [sign the Adobe CLA](http://opensource.adobe.com/cla.html)! Once signed, close and re-open your pull request to run the check again.\n\n If you are an Adobe employee, you do not have to sign the CLA. Instead [add yourself to this GitHub Org](https://git.corp.adobe.com/OpenSourceAdvisoryBoard/handbook/blob/master/GitHub-Adobe-Org-Management.md#request-access-to-our-adobe-github-org) to be recognized as an Adobe Employee.\n\n If you have any questions, contact Adobe\'s Open Source Office by mentioning them on the pull request with **@adobe/open-source-office** or via email <grp-opensourceoffice@adobe.com>.'
```
Comment (filmaj) [inline checker/checker.js]: Should we do "Instead add yourself to this GitHub Org", or "Instead add yourself to the Adobe GitHub Org" ?

Comment (stevengill) [reply checker/checker.js]: sure. We could do that. And update the handbook instructions to tell them to set their membership to public in the org

**Comment by stevengill** [comment]
Updated the wording again

File: checker/checker.js (hunk 1 above)
Comment (filmaj) [reply checker/checker.js]: Hmmm, actually, that's a good point... i forgot about all that.

Comment (filmaj) [reply checker/checker.js]: Lol now I'm second guessing my suggestion.

Comment (filmaj) [reply checker/checker.js]: Sorry! 🙈 just do what you think is right. Now I think your original seems perfectly fine.

**Pull request merged**